portfolio/src/main/webapp/index.html

```diff
@@ -29,6 +29,14 @@ <h1>Anthony Zhu</h1>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomQuote()">Click me!</button>
       <div id="quote-container"></div>
+      <form action="/data" method="POST">
+
+      <p>Leave a comment for me:</p>
```

Comment:
should this be < label > instead? (actual question, it's been a while since I've done this!)

Reply:
I think so because they are associated with each other, but I don't think it makes a difference in this case because we do not need that association for any code. I kept it as <p> because the example website used <p> for the text above the text field.

Reply:
Acknowledged, thank you for looking into this!